datarockets-style.gemspec

```diff
@@ -28,10 +28,9 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rubocop", "~> 0.65.0"
-  spec.add_dependency "rubocop-rspec", "~> 1.32.0"
+  spec.add_dependency "rubocop", "~> 0.72.0"
```

Comment:
I think we'll need to freeze this version in the future cause usual guys do not worry much about backward compatibility of versions (there is a not about it in the gem doc)